### ISSUE TITLE
Enhancer Windfury Weapon

### ISSUE THREAD

**Issue by Nashxzz** [SunwellTracker/issues#3092]
''WATCH LATEST POST''

'Decription: Windfury weaponenchant on Enhance isnt Working right. there Were some Games in Arena Where i did like 30 Autohits and the enchant ditnt even proc once.

How it works:
 I tested on a dummie with Windfury glyph that grants 2% extra procchance: 
1000 Autohits with 2 Weps with same Wep speed.
Mainhandenchant: Windfury, Offhand: Flametongue. So i Did 500 Autohits with Windfury enchant.
Enchant procced 77 Times so its a proccrate of arround 15%. 

How it should work: 
Weaponenchant proccrate is 20% + 2% with Glyph.

**Comment by Helleia** [comment]
There is itnernal cooldownn on this spell. 
https://www.wowhead.com/forums&topic=33843

**Comment by Nashxzz** [comment]
the internal cooldown should not affect the percentage

**Comment by Helleia** [comment]
DId you wait 3 seconds before each hit? As 1000 attacks with 2.6 average sham wep and haste. Your WF wep should hit no slower than once each seconds. So you will have way more attacks than WF is able to handle. The only way to test this is with 1 weapon with WF and you attack--- wait  at least 3 seconds--- attack again. This is the only accurate way to check procs on internal cooldown spells ....

**Comment by Nashxzz** [comment]
Tested it and its fine thank you.
I just tested again on my mate and we got it.
My mate is Pally and we tested with Sacred shield buff.

How it Works:
We didnt got a single Procc while he absorb the autohits. Same on Priest and other absorbs.
Thats explaining why i didnt get proccs against mages or Priests or pallys with Sacred shield.

How it should work:
Every single hit can procc windfury attack even if its absorbed.
